docs/features/pouch_with_gpu.md

```diff
@@ -0,0 +1,36 @@
+# PouchContainer with GPU
+
```

Comment:
Before the graph, I think we should add a general introduction of the usage of PouchContainer with GPU.

Reply:
done